cmd/cloud-controller-manager/main.go

```diff
@@ -65,7 +65,7 @@ func main() {
 
 	s, err := options.NewCloudControllerManagerOptions()
 	if err != nil {
-		klog.Fatalf("unable to initialize command options: %v", err)
+		klog.ErrorS("unable to initialize command options: %v", err)
```

Comment:
Please review https://kubernetes.io/blog/2020/09/04/kubernetes-1-19-introducing-structured-logs/. Also err should be the first parameter to ErrorS